nengo_dl/neuron_builders.py

```diff
@@ -132,8 +132,20 @@ def __init__(self, ops, signals, config):
                 signals.dtype,
             )
 
+        if all(getattr(op.neurons, "negative_slope", 0) == 0 for op in ops):
+            self.negative_slope = None
+        else:
+            self.negative_slope = signals.op_constant(
+                [op.neurons for op in ops],
+                [op.J.shape[0] for op in ops],
+                "negative_slope",
+                signals.dtype,
+            )
+
     def _step(self, J):
         out = tf.nn.relu(J)
+        if self.negative_slope is not None:
```

Comment:
Just to be sure, this `_step` function just sets up the computation right, it's not what gets run on each timestep? So we're not evaluating the if statement on each step, we're just adding some TF ops if this statement is true?

Reply:
Yep that's correct